core/conn/odbc/src/odbc/nsksrvr/SrvrMain.cpp

```diff
@@ -246,8 +246,20 @@ catch(SB_Fatal_Excep sbfe)
 	myProcName = proc_info.process_name;
 
 	char logNameSuffix[32];
-	sprintf( logNameSuffix, "_%d_%d.log", myNid, myPid );
-	CommonLogger::instance().initLog4cxx("log4cxx.trafodion.masterexe.config", logNameSuffix);
+        const char *lv_configFileName = "log4cxx.trafodion.sql.config";
```

Comment:
Thanks for keeping the old behavior as an option. That will make debugging easier when one wishes to attend to just one process.